packages/forma-36-website/src/pages/components/select-field/index.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: 'Select Field'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Select Field is used for choosing a single option from a list of options.
```

Comment:
It's worth mentioning that this is a composition of components - Select, FormLabel, ValidationMessage, etc.